plugins/MauticCrmBundle/Api/SalesforceApi.php

```diff
@@ -520,7 +520,17 @@ private function analyzeResponse($response, $isRetry)
             }
 
             foreach ($response as $lineItem) {
-                if (is_array($lineItem) && !empty($lineItem['errorCode']) && $error = $this->processError($lineItem, $isRetry)) {
+                if (!is_array($lineItem)) {
+                    continue;
+                }
+                $lineItemForInvalidSession              = $lineItem;
+                $lineItemForInvalidSession['errorCode'] = 'INVALID_SESSION_ID';
+                if (!empty($lineItemForInvalidSession['message']) && str_contains($lineItemForInvalidSession['message'], '"errorCode":"INVALID_SESSION_ID"') && $error = $this->processError($lineItemForInvalidSession, $isRetry)) {
```

Comment:
`str_contains` was introduced in PHP 8. Mautic supports PHP 7.4 so this cannot be used.

Reply:
you are right, replaced it with strpos